content/develop/concepts/configuration/theming.md

```diff
@@ -3,107 +3,26 @@ title: Theming
 slug: /develop/concepts/configuration/theming
 ---
 
-# Theming
+# Theming overview
 
-In this guide, we provide examples of how Streamlit page elements are affected
-by the various theme config options. For a more high-level overview of
-Streamlit themes, see the Themes section of the
-[main concepts documentation](/get-started/fundamentals/additional-features#themes).
+In this guide, we provide an overview of theming and visual customization of Streamlit apps. Streamlit themes are defined using configuration options, which are most commonly defined in a `.streamlit/config.toml` file. For more information on setting configuration options, see [Working with configuration options](/develop/concepts/configuration/options).
 
-Streamlit themes are defined using regular config options: a theme can be set
-via command line flag when starting your app using `streamlit run` or by
-defining it in the `[theme]` section of a `.streamlit/config.toml` file. For
-more information on setting config options, please refer to the
-[Streamlit configuration documentation](/develop/concepts/configuration).
+The following options can be set once for your whole app:
 
-The following config options show the default Streamlit Light theme recreated
-in the `[theme]` section of a `.streamlit/config.toml` file.
+- **Base color scheme**: Set your custom theme to inherit from Streamlit's light or dark theme.
+- **Font size**: Set the base font size for your app.
 
-```toml
-[theme]
-primaryColor="#FF4B4B"
-backgroundColor="#FFFFFF"
-secondaryBackgroundColor="#F0F2F6"
-textColor="#31333F"
-font="sans serif"
-```
+The following options can be configured separately for the main body of your app and the sidebar:
 
-Let's go through each of these options, providing screenshots to demonstrate
-what parts of a Streamlit app they affect where needed.
+- **Font**: Set the font family for body text, headers, and code.
+- **Text color**: Set the color of body text and links.
+- **Accent color**: Set the color of interactive elements and highlights.
```

Comment:
Sounds a bit weird to say accent color here if it's called primary color everywhere else.